src/modules/dataFetching/components/GenericTableWithData.tsx

```diff
@@ -336,15 +283,15 @@ const GenericTableWithData = <
 
   const noDataValue = useMemo(() => {
     if (typeof noData === 'function') return noData(filterValues);
-    if (!showFilters) return noData;
+    if (!showTopToolbar) return noData;
```

Comment:
The intent here is to return if the table doesn't have filtering. If it has filtering, then we show a different message below to make sure the user realizes there is a filter (pretty rudimentary currently..). So, this should probably be `if (!filters) return noData;`.